src/opnsense/www/api.php

```diff
@@ -1,56 +1,37 @@
 <?php
 
-try {
-    /**
-     * Read the configuration
-     */
-    $config = include __DIR__ . "/../mvc/app/config/config.php";
-
-    /**
-     * Read auto-loader
-     */
-    include __DIR__ . "/../mvc/app/config/loader.php";
-
-    /**
-     * Read services
-     */
-    include __DIR__ . "/../mvc/app/config/services_api.php";
-
-    /**
-     * Handle the request
-     */
-    $application = new \Phalcon\Mvc\Application($di);
-
-    echo $application->handle($_SERVER['REQUEST_URI'])->getContent();
-} catch (\Error | \Exception $e) {
-    $unbuffered = strpos($e->getMessage(), 'ob_end_clean') !== false;
-
-    if (!is_a($e, 'OPNsense\Base\UserException') && !$unbuffered) {
-        error_log($e);
-    }
-
-    if ($unbuffered) {
-        /**
-         * if buffer has been deleted already, we must assume the implementation
-         * has already sent headers and we cannot send them again.
-         */
-        return;
+function error_output($http_code, $e,  $user_message)
+{
+    $response = [];
+    if (OPNsense\Core\Config::getInstance()->object()->system->deployment != 'development'){
```

Comment:
Checking existence of `/var/run/development` is quicker (and a workaround for pre-config-readability in legacy parts)

Reply:
thanks, I'll change that one